### ISSUE TITLE
Provide reference noise maps for unit testing

### ISSUE THREAD

**Issue by zonca** [simonsobs/mapsims#10]
Thanks @damonge for volunteering to provide those, I implemented a quick unit test for now, but I would be more confident if I have reference maps computed independently by you using another code.

You can attach to this issue once you have created them

> > > @damonge can you also please generate 2 low resolution (nside 16 or 32) test maps (1 SAT channel and 1 LAT channel) with a predefined random seed for unit testing at attach them to this Pull Request?
> > 
> > 
> > Sure, can do. How would you want me to do that exactly? There's significant overlap between the code we used for the paper and this code, so it should be no wonder if both agree exactly (if the point is to validate the current code).
> 
> the plan is to validate this code and make it more difficult to break it unintentionally in the future.
> the easiest test would be 1 SAT channel and 1 LAT channel with all default parameters (or what you think is the most "standard" configuration as long as you specify it) at NSIDE 16 with a fixed numpy random seed.

as for scanning strategy, you can choose either classical or opportunistic

_Originally posted by @zonca in https://github.com/simonsobs/mapsims/pull/5#issuecomment-452011344_

**Comment by damonge** [comment]
@zonca I'll do ASAP. Sorry I haven't had time yet, but I'll try to get to it tomorrow.

**Comment by damonge** [comment]
@zonca : here are the maps and the script I used to produce them.

Two possible issues with the current implementation I just realized:
- I think the noise calculator returns power spectra for l>=2, but synfast needs you to pass power spectra starting at l=0. I believe the current code doesn't correct for this.
- I think the noise calculator already returns power spectra in CMB temperature units. I think the current code assumes that they are antenna temperature units and corrects for it.

[mapsims_benchmark.zip](https://github.com/simonsobs/mapsims/files/2768348/mapsims_benchmark.zip)
